### ISSUE TITLE
placeholder="Place your ad banner code here." does not exist in Site edit.

### ISSUE THREAD

**Issue by Sweepr** [nZEDb/nZEDb#1597]
<a href='http://postimg.org/image/qalukt4cj/' target='_blank'><img src='http://s13.postimg.org/qalukt4cj/image.jpg' border='0' alt="hf" /></a>

**Comment by niel** [comment]
I don't understand your point.

**Comment by zetas** [comment]
That's very strange, it should be there. What branch are you running?

**Comment by Sweepr** [comment]
@niel 
Yes, the text box is missing indeed, the other two are there.

@zetas 
At this moment i'm running the Master branch.

**Comment by niel** [comment]
Looks like this is caused by ad blockers. I'll see about changing the way it is identified to prevent this.

**Comment by Sweepr** [comment]
@niel 
Yes indeed it was blocked by adblocker, confusing because the others were showing the box with adblock on.

**Comment by zetas** [comment]
@Sweepr The reason for that is probably because the ID for that textarea is "adheader". If I were writing an adblocker, i'd probably block that ID too :P

**Comment by Sweepr** [comment]
@zetas 
Yes, after removing the ID for the DIV the area started showing again.

**Comment by niel** [comment]
Removing the ID alone isn't a solution. The entire code path for the entry has to be changed, it's on my TODO list but frankly it's not a priority as disbling your adblocker for ZED sites removes the problem.

**Comment by Sweepr** [comment]
@niel 

I understand, i'll close this
